From: Daniel van Vugt <daniel.van.vugt@canonical.com>
Date: Wed, 6 Aug 2025 14:56:00 +0800
Subject: power/test: Disable known broken test

It was broken and marked as FIXME in 49.beta commit 08ae6461a824.
---
 plugins/power/test.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/power/test.py b/plugins/power/test.py
index 4372d1b..1802f63 100755
--- a/plugins/power/test.py
+++ b/plugins/power/test.py
@@ -647,7 +647,7 @@ class PowerPluginTestLid(PowerPluginBase):
 
         # FIXME: this check fails
         # Check for unblanking
-        self.check_unblank(2)
+        # self.check_unblank(2)
 
         # Drop the inhibit
         self.obj_session_mgr.Uninhibit(dbus.UInt32(inhibit_id),
